packages/ag-charts-core/src/utils/numbers.ts

```diff
@@ -1,4 +1,8 @@
 export function clamp(min: number, value: number, max: number) {
+    if (value == null) {
```

Comment:
I think we should not do this check here. Do we know what was passing in nil values?

Reply:
I've updated this to `throw Error()` in this situation, and found one spot in `coneFunnelUtil.ts` that was causing issues. This was showing up as a problem more widely before, so perhaps your suggestion for `text.ts` improved things too.

Reply:
Revert this change in favor of defaulting using `??` at the call site for now.